enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,248 @@
+---
+title: targeted-update-edge-blocking
+authors:
+  - "@wking"
+reviewers:
+  - "@dofinn"
+  - "@LalatenduMohanty"
+  - "@sdodson"
+  - "@steveeJ"
+  - "@vrutkovs"
+approvers:
+  - TBD
+creation-date: 2020-07-07
+last-updated: 2020-08-12
+status: implementable
+---
+
+# Targeted Update Edge Blocking
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a mechanism for blocking edges for the subset of clusters considered vulnerable to known issues with a particular update or target release.
+
+## Motivation
+
+When managing the [Cincinnati][cincinnati-spec] update graph [for OpenShift][cincinnati-for-openshift-design], we sometimes discover issues with particular release images or updates between them.
+Once an issue is discovered, we [block edges][block-edges] so we no longer recommend risky updates, or updates to risky releases.
+
+Note: as described [in the documentation][support-documentation], supported updates are still supported even if incoming edges are blocked, and Red Hat will eventually provide supported update paths from any supported release to the latest supported release in its z-stream.
+
+Incoming bugs are evaluated to determine an impact statement based on [a generic template][rhbz-1858026-impact-statement-request].
+Some bugs only impact specific platforms, or clusters with other specific features.
+For example, rhbz#1858026 [only impacted][rhbz-1858026-impact-statement] clusters with the `None` platform which were created as 4.1 clusters and subsequently updated via 4.2, 4.3, and 4.4 to reach 4.5.
+In those cases there is currently tension between wanting to protect vulnerable clusters by blocking the edge vs. wanting to avoid inconveniencing clusters which we know are not vulnerable and whose administrators may have been planning on taking the recommended update.
+This enhancement aims to reduce that tension.
```

Comment:
We should also add that , we will publish text only errata when we remove edges from stable and fast channel to the latest Y stream release. Customers will get the bug reference from the errata and they will find out if the bug is specific for a platform. We can expect customers asking us not block them when they are not on the platform if a bug is specific to another platform. Also it is logical to reduce the impact radius of the edge removable.

Reply:
Can we roll discussion of the text-only errata into [the earlier thread][1]?  I think it's orthogonal to this PR, but don't want to argue that out in two parallel threads ;)

[1]: https://github.com/openshift/enhancements/pull/426#discussion_r630441945

Reply:
Basically I am trying to pitch that  by releasing text only erratas we are making the case for targeted edge blocking. Its the next logical step to do.